src/libPMacc/include/simulationControl/SimulationHelper.hpp

```diff
@@ -40,6 +40,13 @@
 #include <iomanip>
 #include <fstream>
 
+#ifdef SCOREP_USER_ENABLE
```

Comment:
please revert the changes in this single file, we can not merge those to the mainline since they are too specific.